Finjector.Web/ClientApp/src/components/Shared/ChartList.tsx

```diff
@@ -70,24 +70,26 @@ const ChartList = (props: Props) => {
               <FontAwesomeIcon icon={faUpRightFromSquare} />
             </Link>
           </h2>
-          {teamGroup.folders.map((folder) => (
-            <div className="folder-chart-wrapper" key={folder.id}>
-              <h3>
-                <Link
-                  className="no-link-style d-flex vertical-align-top"
-                  to={`/teams/${teamGroup.team.id}/folders/${folder.id}`}
-                >
-                  {folder.name}
-                  <FontAwesomeIcon icon={faUpRightFromSquare} />
-                </Link>
-              </h3>
-              <ul className="list-group">
-                {folder.coas.map((chart) => (
-                  <ChartListItem key={chart.id} chart={chart} />
-                ))}
-              </ul>
-            </div>
-          ))}
+          <div className="team-chart-wrapper" key={teamGroup.team.id}>
```

Comment:
@cydoval I think this key id could be the same as the folder id?
So maybe prefix with team text?